apps/cowswap-frontend/src/modules/orderProgressBar/pure/LottieContainer.tsx

```
@@ -0,0 +1,65 @@
+`
+
+interface FullSizeLottieProps {
+  animationData: unknown
+  loop?: boolean
+  autoplay?: boolean
+}
+
+export function FullSizeLottie({ animationData, loop = true, autoplay = true }: FullSizeLottieProps): ReactNode {
+  const lottieRef = useRef<HTMLDivElement>(null)
+
+  useEffect((): (() => void) | void => {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)